_data/projects/opencensus-go.yml

```diff
@@ -0,0 +1,22 @@
+name: opencensus-go
+desc: A stats collection and distributed tracing framework
+site: https://opencensus.io
+tags:
+- trace
+- go
+- cloud
+- stats
+- monitoring
+- opencensus
+- tracing
+- distributed-tracing
+- dapper
+- metrics
+- prometheus
+- zipkin
+- stackdriver
+- signalfx
+- application insights
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/census-instrumentation/opencensus-node
```

Comment:
Each of these links should go to a curated list of issues that are ideal for new contributors. We use `up-for-grabs` by convention, but you can use a different label - as long as `name` and `link` are consistent.

Reply:
@SergeyKanzhelev Here, please have a look at this.